tools/ci_build/build.py

```diff
@@ -276,8 +276,12 @@ def parse_arguments():
         choices=["armeabi-v7a", "arm64-v8a", "x86", "x86_64"],
         help="Specify the target Android Application Binary Interface (ABI)")
     parser.add_argument("--android_api", type=int, default=27, help='Android API Level, e.g. 21')
-    parser.add_argument("--android_sdk_path", type=str, help='Path to the Android SDK')
-    parser.add_argument("--android_ndk_path", default="", help="Path to the Android NDK")
+    parser.add_argument(
+        "--android_sdk_path", type=str, default=os.environ.get("ANDROID_HOME", ""),
+        help="Path to the Android SDK")
+    parser.add_argument(
+        "--android_ndk_path", type=str, default=os.environ.get("ANDROID_NDK_HOME", ""),
+        help="Path to the Android NDK")
     parser.add_argument("--android_cpp_shared", action="store_true",
```

Comment:
nit: do we need to spread this over 3 lines or can we just use 2 (and I find slightly more readable as it's easier to skim between the 'add_argument' calls when there's just whitespace under each one)

```
    parser.add_argument("--android_sdk_path", type=str, default=os.environ.get("ANDROID_HOME", ""),
                        help="Path to the Android SDK")
    parser.add_argument("--android_ndk_path", type=str, default=os.environ.get("ANDROID_NDK_HOME", ""),
                        help="Path to the Android NDK")
```